try:
    a=5
    b=a/0
    print(b)
    prin(c)
except Exception as e:
    print(e)
finally:
    print("Done.")
